package.json

```diff
@@ -41,9 +41,10 @@
     "watch": "preconstruct watch",
     "dev": "preconstruct dev",
     "demo:dev": "yarn workspace demo dev",
-    "test": "jest",
+    "test:unit": "jest",
     "test:cov": "jest --coverage",
     "test:ts": "tsc --noEmit",
+    "test": "yarn test:unit && cd ./packages/parallax && yarn test",
```

Comment:
can we remove the need for `cd` bringing `cypress` to the top level?

Also let's specifically name this `test:e2e` and then test can be `yarn test:unit && yarn test:e2e`

Reply:
I've renamed the test scripts and also got rid of `cd` by using `yarn workspace @react-spring/parallax test`. Was `cd` the issue or is it that `/cypress` should be in the root?

Reply:
I was just thinking in general that the cypress set up should be at the root of the mono repo 👍🏼

Reply:
Got it! Moved `cypress` folder and `cypress.json` to root.